pkg/policy/rule.go

```diff
@@ -327,6 +329,37 @@ func computeResultantCIDRSet(cidrs []api.CIDRRule) []api.CIDR {
 	return allResultantAllowedCIDRs
 }
 
+func cidrsToIPNets(cidrs []api.CIDR) (v4Nets, v6Nets []*net.IPNet) {
```

Comment:
Unused function?

Reply:
Unused, but I left it around as it could prove useful.